descriptions/api.github.com/api.github.com.json

```diff
@@ -7457,7 +7358,7 @@
       "get": {
         "summary": "Get GitHub Actions permissions for an organization",
         "description": "Gets the GitHub Actions permissions policy for repositories and allowed actions in an organization.\n\nYou must authenticate using an access token with the `admin:org` scope to use this endpoint. GitHub Apps must have the `administration` organization permission to use this API.",
-        "operationId": "actions/actions-policies/get-github-actions-permissions-organization",
+        "operationId": "actions/get-github-actions-permissions-organization",
```

Comment:
this resolves https://github.com/github/rest-api-description/issues/272, thank you @dinonuggies1